Client/Source/Main.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+
+#include <QtCore/QCoreApplication>
+
+#include "Client.h"
+
+
+int main(int argc, char *argv[])
+{
+    QCoreApplication a(argc, argv);
+
+    auto& printer = printer::Printer::getInstance();
+    printer(std::cout, "Console client for connecting to Fanuc M-20iA v 0.1\n\n");
+
+    constexpr int  kServerPort = 8888;
+    constexpr int  kServerReceivingPort = 9998;
+    constexpr int  kServerSendingPort = 9999;
+    constexpr char kServerIP[] = "192.168.0.100";
+
+    // Make sure that you use right client: 1 - debug, 2 - for layer, 3 - for robot.
+    ///vasily::Client client(kServerReceivingPort, kServerSendingPort, kServerIP,
+    ///                      vasily::Client::WorkMode::STRAIGHTFORWARD);
+
+    vasily::Client client(kServerPort, kServerIP, vasily::Client::WorkMode::INDIRECT);
+
+    ///vasily::Client client{};
```

Comment:
Do you need 26 line?

Reply:
Hmmm... Do you read my [THIS COMMENT](https://github.com/Vasar007/RoboticsSystem/pull/32/files/b5d9abd103e0fea6780f344b7c0462b8a1c6c2e3#diff-b6398eb975ba7dbe798be672408eaa53R20)???

Will there be comments on the real code, not my notes in the code?